test/e2e/e2e.yml

```diff
@@ -59,7 +59,7 @@ spec:
         app: open-feature-e2e-test
       annotations:
         openfeature.dev: "enabled"
-        openfeature.dev/featureflagconfiguration: "end-to-end-test"
+        openfeature.dev/featureflagconfiguration: "open-feature-operator-system.end-to-end-test"
```

Comment:
Can we use `/` as is standard in k8s? e.g. `namespace/service`

Reply:
ive introduced this change in both PRs